source/configuration/modules/imklog.rst

```diff
@@ -1,34 +1,143 @@
+behavior, do not specify $klogConsoleLogLevel in the configuration
+file. Note that this is a global parameter. Each time it is changed,
+the previous definition is re-set. The one activate will be that one
+that is active when imklog actually starts processing. In short
+words: do not specify this directive more than once!
 
-.. function:: $KLogInternalMsgFacility <facility>
+**Linux only**, ignored on other platforms (but may be specified)
 
-   The facility which messages internally generated by imklog will
-   have. imklog generates some messages of itself (e.g. on problems,
-   startup and shutdown) and these do not stem from the kernel.
-   Historically, under Linux, these too have "kern" facility. Thus, on
-   Linux platforms the default is "kern" while on others it is
-   "syslogd". You usually do not need to specify this configuration
-   directive - it is included primarily for few limited cases where it
-   is needed for good reason. Bottom line: if you don't have a good idea
-   why you should use this setting, do not touch it.
 
-.. function:: $KLogPermitNonKernelFacility off
+ParseKernelTimestamp
+^^^^^^^^^^^^^^^^^^^^
 
-   At least under BSD the kernel log may contain entries with
-   non-kernel facilities. This setting controls how those are handled.
-   The default is "off", in which case these messages are ignored.
-   Switch it to on to submit non-kernel messages to rsyslog processing.
+.. csv-table::
+   :header: "type", "default", "mandatory", "|FmtObsoleteName| directive"
+   :widths: auto
+   :class: parameter-table
+
+   "binary", "off", "no", "``$klogParseKernelTimestamp``"
+
+If enabled and the kernel creates a timestamp for its log messages,
+this timestamp will be parsed and converted into regular message time
+instead to use the receive time of the kernel message (as in 5.8.x
+and before). Default is 'off' to prevent parsing the kernel timestamp,
+because the clock used by the kernel to create the timestamps is not
+supposed to be as accurate as the monotonic clock required to convert
+it. Depending on the hardware and kernel, it can result in message
+time differences between kernel and system messages which occurred at
+same time.
+
+
+KeepKernelTimestamp
+^^^^^^^^^^^^^^^^^^^
+
+.. csv-table::
+   :header: "type", "default", "mandatory", "|FmtObsoleteName| directive"
+   :widths: auto
+   :class: parameter-table
+
+   "binary", "off", "no", "``$klogKeepKernelTimestamp``"
+
+If enabled, this option causes to keep the [timestamp] provided by
+the kernel at the begin of in each message rather than to remove it,
+when it could be parsed and converted into local time for use as
+regular message time. Only used, when $klogParseKernelTimestamp is
+on.
+
+
+LogPath
+^^^^^^^
+
+.. csv-table::
+   :header: "type", "default", "mandatory", "|FmtObsoleteName| directive"
+   :widths: auto
+   :class: parameter-table
+
+   "word", "none", "no", "``$klogpath``"
+
+???
```

Comment:
I found a parameter which wasn't documented until now and needs to be looked at.